<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CarMainWindow</class>
 <widget class="QMainWindow" name="CarMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CarMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>781</width>
      <height>365</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="StartTab">
     <attribute name="title">
      <string>Start</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>220</y>
        <width>171</width>
        <height>91</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="autoStartButton">
         <property name="font">
          <font>
           <family>Bitstream Charter</family>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Auto start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="manualStartButton">
         <property name="font">
          <font>
           <family>Bitstream Charter</family>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Manual start</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QListView" name="listView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>261</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>311</width>
        <height>301</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="startLabel">
         <property name="text">
          <string>Start:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="minLineEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="stopLabel">
         <property name="text">
          <string>Stop:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="maxLineEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="unitLabel">
         <property name="text">
          <string>Unit:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="unitComboBox"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="settingsTab">
     <attribute name="title">
      <string>Settings</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>781</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="view1Button">
       <property name="text">
        <string>View1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="view2Button">
       <property name="text">
        <string>View2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="view3Button">
       <property name="text">
        <string>View3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="view4Button">
       <property name="text">
        <string>View4</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
